src/dra/843B0.c

```diff
@@ -2681,8 +2681,8 @@ void func_80129864(Entity* self) {
             prim->drawMode |= DRAW_HIDE;
             break;
         case 1:
-            temp_u = D_800B0846[self->animCurFrame * 2];
-            temp_v = D_800B0846[self->animCurFrame * 2 + 1];
+            temp_u = D_800B0844[self->animCurFrame * 2 + 2];
```

Comment:
Could you explain this change?

Reply:
The data in this section is weird:

```
glabel D_800B0830
/* 10830 800B0830 0080FDFF */ .word 0xFFFD8000
/* 10834 800B0834 0080FDFF */ .word 0xFFFD8000
/* 10838 800B0838 0080FDFF */ .word 0xFFFD8000
.size D_800B0830, . - D_800B0830

glabel D_800B083C
/* 1083C 800B083C 00000000 */ .word 0x00000000
/* 10840 800B0840 0040FFFF */ .word 0xFFFF4000
/* 10844 800B0844 */ .byte 0x00
/* 10845 800B0845 */ .byte 0xC0
.size D_800B083C, . - D_800B083C

glabel D_800B0846
/* 10846 800B0846 */ .byte 0x00
.size D_800B0846, . - D_800B0846

glabel D_800B0847
/* 10847 800B0847 */ .byte 0x00
/* 10848 800B0848 80808090 */ .word 0x90808080
/* 1084C 800B084C 80A080B0 */ .word 0xB080A080
/* 10850 800B0850 A080A090 */ .word 0x90A080A0
/* 10854 800B0854 A0A0A0B0 */ .word 0xB0A0A0A0
.size D_800B0847, . - D_800B0847

glabel D_800B0858
/* 10858 800B0858 */ .short 0x03C0
/* 1085A 800B085A */ .short 0x0440
/* 1085C 800B085C */ .short 0x0340
/* 1085E 800B085E */ .short 0x04C0
.size D_800B0858, . - D_800B0858
```

It's possible the extra bytes are actually part of `D_800B083C` making it size 3 to mirror `D_800B0830`?

The [PSP version](https://decomp.me/scratch/g2m4c) seems to be:

```c
temp_u = D_800B0846[(s2->animCurFrame - 1) * 2];
temp_v = D_800B0846[(s2->animCurFrame - 1) * 2 + 1];
```

Reply:
The PSP version matches as far as I can tell?             
```
  temp_u = D_800B0848[(self->animCurFrame - 1) * 2];
  temp_v = D_800B0848[(self->animCurFrame - 1) * 2 + 1];
```
https://decomp.me/scratch/HJsV3

Reply:
Ah! That would make `0x00000C00` part of the previous array, which translates to `FIX(0.75)`.

https://github.com/Xeeynamo/sotn-decomp/blob/3feb5dff994c7bd0c3aa5570280581cb915598ec/src/dra/d_10798.c#L10-L12

I will update.